src/modules/ekf2/EKF/covariance.cpp

```diff
@@ -201,14 +201,15 @@ void Ekf::predictCovariance(const imuSample &imu_delayed)
 #if defined(CONFIG_EKF2_WIND)
 
 	// wind vel: add process noise
-	if (!_external_wind_init) {
-		float wind_vel_nsd_scaled = math::constrain(_params.wind_vel_nsd, 0.f, 1.f)
-					    * (1.f + _params.wind_vel_nsd_scaler * fabsf(_height_rate_lpf));
-		float wind_vel_process_noise = sq(wind_vel_nsd_scaled) * dt;
-
-		for (unsigned index = 0; index < State::wind_vel.dof; index++) {
-			const unsigned i = State::wind_vel.idx + index;
-			P(i, i) = fminf(P(i, i) + wind_vel_process_noise, sq(_params.initial_wind_uncertainty));
+	float wind_vel_nsd_scaled = math::constrain(_params.wind_vel_nsd, 0.f, 1.f)
```

Comment:
Side note, we could apply these constraints centrally once instead of every iteration.